explainer.md

```diff
@@ -1,3 +1,122 @@
+ * an absolute capture timestamp, which is stamped by the original capturer, and is supposed to be received untouched by the end receivers.
+ * an estimated clock offset with respect to the capturer's clock, which is supposed to be read and updated by every intermediate sender.
+
+With the absolute capture timestamps, end receivers can accurately measure the audio-video synchronization performance. With the `estimated clock offset`, which gets updated at each intermediate hop, end receivers can estimate their respective clock offset with respect to the capturer's clock, and then together with the absolute capture timestamp, measure the end-to-end delay. 
+
+The absolute capture time RTP header extension is defined [here](https://github.com/webrtc/webrtc-org/blob/gh-pages/experiments/rtp-hdrext/abs-capture-time/index.md).
+
+The two new data fields in RTCRtpContributingSource are to surface the two data fields in the absolute capture time RTP header extension.
+
+#### Goals
+
+ * Facilitate calculation of audio video synchronization performance in real-time communication systems.
+ * Facilitate calculation of end-to-end delay in real-time communication systems, particularly those that involve intermediate stream regenerators.
+
+##### Non-goals
+
+This proposal does not aim for improving the accuracy of end-to-end delay measurement in real-time communication systems that can already measure it based on RTCP based schemes.
+
+#### [API 1]: captureTimestamp
+
+This specification adds captureTimestamp, type of DOMHighResTimeStamp, to the RTCRtpContributingSource dictionary. This surfaces the absolute capture timestamp in the absolute capture time RTP header extension, when it is present or can be extrapolated from previously received data, for the last rendered audio or video frame. It can be used for measuring audio video synchronization performance as illustrated in the following exemplar code:
+
+    [receiverAudio, receiverVideo] = peerconnection.getReceivers();
+
+    latestCaptureTimestampsAudio = receiverAudio.getSynchronizationSources()[0].captureTimeStamp;
+    latestCaptureTimestampsVideo = receiverVideo.getSynchronizationSources()[0].captureTimeStamp;
+
+    avSynchronization = latestCaptureTimestampsAudio - latestCaptureTimestampsVideo;
+
+#### [API 2]: senderCaptureTimeOffset
+
+This specification also adds senderCaptureTimeOffset, type of DOMHighResTimeStamp, to the RTCRtpContributingSource dictionary. In this context, the sender refers to the system that directly sends RTP and RTCP packets to the receiver, and thus the sender-receiver path only represents the "last hop" in a system that involves intermediate stream regenerators.
+
+An exemplar code to use captureTimestamp and senderCaptureTimeOffset to calculate end-to-end delay follows:
+
+    receiver = peerconnection.getReceivers()[0];
+    csrc = receiver.getSynchronizationSources()[0];
+    latestCaptureTimestamps = csrc.captureTimeStamp;
```

Comment:
latestCaptureTimestamps -> latestCaptureTimestamp (there is only one)

Reply:
Thanks!